source/upgrade-guide/troubleshooting.rst

```diff
@@ -1,50 +1,163 @@
+   :description: This section contains common issues that might occur when upgrading central components and provides steps to resolve them.
 
 Troubleshooting
 ===============
 
-This section collects common issues that might occur when upgrading the Wazuh central components and provides steps to resolve them.
+This section contains common issues that might occur when upgrading the Wazuh central components and provides steps to resolve them.
+
+Wazuh-DB backup restoration
+---------------------------
+
+Wazuh by default performs automatic backups of the ``global.db`` database. These snapshots may be useful to recover critical information such as agent keys, agent synchronization information, and FIM event data among others. :doc:`Wazuh-DB </user-manual/reference/daemons/wazuh-db>` will restore the last backup available in case of failure during the upgrade. If this process also fails, the restoration must be done manually.
+
+Manual restoration process
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Stop the Wazuh manager.
+
+   .. tabs::
+
+      .. tab:: Systemd
+
+         .. code-block:: console
+
+            # systemctl stop wazuh-manager
+
+      .. tab:: SysV init
+
+         .. code-block:: console
+
+            # service wazuh-manager stop
+
+#. Locate the backup to restore. It is stored in ``/var/ossec/backup/db/`` with a name format similar to ``global.db-backup-TIMESTAMP-pre_upgrade.gz``.
+
+   .. note::
+
+      This process is valid for all the backups in the folder. Snapshot names containing the special tag ``pre_upgrade`` were created right before upgrading the Wazuh server. Any other snapshot is a periodic backup created according to the :doc:`backup </user-manual/reference/ossec-conf/wazuh-db-config>` setting.
+
+#. Decompress the backup file. Always use the ``-k`` flag to preserve the original file:
+
+   .. code-block:: console
+
+      # gzip -dk WAZUH_HOME/backup/db/global.db-backup-TIMESTAMP-pre_upgrade.gz
+
+#. Remove the current ``global.db`` database and move the backup to the right location:
+
+   .. code-block:: console
+
+      # rm  WAZUH_HOME/queue/db/global.db
+      # mv  WAZUH_HOME/backup/db/global.db-backup-TIMESTAMP-pre_upgrade WAZUH_HOME/queue/db/global.db
+
+#. Start the Wazuh manager.
+
+   .. tabs::
+
+      .. tab:: Systemd
+
+         .. code-block:: console
+
+            # systemctl start wazuh-manager
+
+      .. tab:: SysV init
+
+         .. code-block:: console
+
+            # service wazuh-manager start
+
+Wazuh dashboard server is not ready yet
+---------------------------------------
+
+This message typically appears right after starting or restarting the Wazuh dashboard. However, it may also indicate one of the following issues:
+
+- The Wazuh dashboard service is encountering an error and repeatedly restarting.
+- The Wazuh dashboard cannot communicate with the Wazuh indexer.
+- The Wazuh indexer service is not running or has encountered an error.
```

Comment:
Use three-space indentation such as `-  ` to enhance readability and prevent misalignments.